get_weather_Utils.py

```diff
@@ -0,0 +1,84 @@
+from decimal import *
+import boto3
+import requests
+from datetime import date, timedelta, datetime
+import formatting_Utils
+
+
+
+def make_url_api(location_id, day_str):
+    url = "https://www.metaweather.com/api/location/" + location_id + "/"+ day_str  + "/"
+    print("API-URL aus Funktion make_url_api: ", url)
+    return url
+
+
+def get_location(number):
+    dynamodb = boto3.resource('dynamodb')
+    table_destination = dynamodb.Table('destination')
+    response = table_destination.scan()
+    data = response['Items']
+    while 'LastEvaluatedKey' in response:
+        response = table_destination.scan(ExclusiveStartKey=response['LastEvaluatedKey'])
+        data.extend(response['Items'])
+    city = response['Items'][number]['city']
+    city_id = response['Items'][number]['city_id']
+    print(city, city_id)
+    return [city, city_id]
+
+
+
+def write_weather_dynamoDB(api_url, location):
+    url = requests.get(api_url)
+    response = url.json()
+
+    dynamodb = boto3.resource('dynamodb')
+
+    table_weather = dynamodb.Table('weather')
+
+
+    counter = 0
+    for item in response:
+        
+        #city in DB
+        # date = applicable_date
+        report = counter
+        id = response[counter]['id']
```

Comment:
why you are not using the item directly? 
```suggestion
        id = item['id']
```
its the same for the other parameters

Reply:
Done 👍🏼